images/opensuse-micro

```diff
@@ -0,0 +1 @@
+opensuse-micro-5be65dadbd70accf45e6dcc29417d853c253fe44af0a4bb4fc71d432adf52cfb.qcow2
```

Comment:
These images need to be build by our bots, please drop these and once the PR is approved we will trigger bots to refresh these images.